scripts/04-test.sh

```diff
@@ -36,4 +36,57 @@ popd || exit 1
 echo "Testing Swagger UI is available"
 curl -I http://$APPLICATION_ENDPOINT/swagger-ui/index.html
 
-# TODO add addition API verification tests
+API_BASE="http://$APPLICATION_ENDPOINT"
+
+echo "Logging into API as EHR"
+EHR_AUTH_TOKEN=$(curl -s -X 'POST' \
+  "$API_BASE/auth/login" \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "username": "userEhr"
+}' | jq -r '.token')
+
+echo "Send Patient Statuses for Disaster 1001"
+echo "Create patient with id"
```

Comment:
Do we want to add the patient Ids here in the echo? The returned result from the curl is our internal ID, but we might want to include the custom id we are sending here.